http/action/src/test/java/io/knotx/databridge/http/action/HttpActionTest.java

```diff
@@ -23,10 +23,15 @@
 import static io.knotx.databridge.http.action.HttpAction.TIMEOUT_TRANSITION;
```

Comment:
It is hard to find what data is logged when log level is:
- ERROR
- INFO

I think that error log level should add log entries only if status code is invalid. In other case the node log should be empty.

When info log level is set, then we log all data always.

When predicate is not passed (such as expected_200_status _code), then an exception is passed. We should check the exception type and try to log response details if available.